proto/attestation/evidence.proto

```diff
@@ -92,10 +89,10 @@ message ApplicationKeys {
 // <https://datatracker.ietf.org/doc/html/rfc9334#name-evidence>
 message Evidence {
   // Layer0 attestation evidence.
-  RootLayerEvidence root_layer_evidence = 1;
+  RootLayerEvidence root_layer = 1;
```

Comment:
I think it's better to keep the `root_layer_evidence` and `root_layer_endorsement` (and `layer_evidences` as well) names to be explcit. Also when we'll create local variables our of them, having exact same `root_layer` names may be confusing.

Reply:
I assumed that it is more concise, and also consistent between evidence, endorsement, ref values. It is just the method name, right? In java, it looks like this:

RootLayerEvidence rootEvidence = evidence.getRootLayer();
LayerEvidence e = evidence.getLayers(i);

Isn't this nicer than repeating the term `evidence` so many times in one line?